tests/DwitTech.NotificationService.WebApi.Tests/Controller/EmailControllerTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using DwitTech.NotificationService.Core.Dtos;
+using DwitTech.NotificationService.Core.Interfaces;
+using DwitTech.NotificationService.Core.Services;
+using DwitTech.NotificationService.Data.Context;
+using DwitTech.NotificationService.Data.Entities;
+using DwitTech.NotificationService.Data.Repository;
+using DwitTech.NotificationService.WebApi.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace DwitTech.NotificationService.WebApi.Tests.Controller
+{
+    public class EmailControllerTest
+    {
+
+        private readonly Mock<IEmailService> _mockService;
+        private readonly EmailController _controller;
+        private readonly IConfiguration _configuration;
+       
+        public EmailControllerTest()
+        {
+            
+            _configuration = new ConfigurationBuilder().AddInMemoryCollection(new Dictionary<string, string> {
+
+                {"GmailInfo:Email", "johnokpo83@gmail.com"},
+                {"GmailInfo:Host", "smtp.gmail.com"},
+                {"GmailInfo:Port", "587"},
+                {"GmailInfo:AppPassword", "fyvdxxvlsvosecwg"},
+
+            }).Build();
+
+        }
+
+
+
+        [Fact]
+       public void SendEmail_ShouldReturn20()
```

Comment:
I think you meant to write 200 here, not 20?

Reply:
> I think you meant to write 200 here, not 20?

Yeah it is meant to be 200 which will return ok